content/en/user-guide/aws/opensearch/index.md

```diff
@@ -334,6 +334,22 @@ Finally create an example index using the following command:
 $ curl -X PUT my-domain.us-east-1.opensearch.localhost.localstack.cloud:4566/my-index
 {{< /command >}}
 
+## Resource Browser
+
+The LocalStack Web Application provides a Resource Browser for managing OpenSearch domains. 
+You can access the Resource Browser by opening the LocalStack Web Application in your browser, navigating to the **Resources** section, and then clicking on **OpenSearch Service** under the **Analytics** section.
+
+<img src="opensearch-resource-browser.png" alt="OpenSearch Resource Browser" title="OpenSearch Resource Browser" width="900" />
+<br>
+<br>
+
+The Resource Browser allows you to perform the following actions:
+
+- **Create Domain**: Create a new OpenSearch domain by clicking on the **Create Domain** button and providing the required details.
+- **View Domain details**: Click on a domain to view its details, such as the domain name, status, endpoint, and configuration.
```

Comment:
The header for this has Details capitalized
```suggestion
- **View Domain Details**: Click on a domain to view its details, such as the domain name, status, endpoint, and configuration.
```


![image](https://github.com/localstack/docs/assets/13120099/7ea1518c-f305-4cb4-983b-98a152aac923)